bundles/SimpleBackendSearchBundle/src/Command/SearchBackendReindexCommand.php

```diff
@@ -46,62 +45,59 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     {
         // clear all data
         $db = \Pimcore\Db::get();
-        $db->executeQuery('TRUNCATE `search_backend_data`;');
```

Comment:
mhh, wondering if this change may result in residual entries accumulating over time, potentially leading to inconsistencies or unnecessary clutter when the PRE_DELETE event somehow doesn't triggered

Maybe add a flag as a variant to re-index from scratch?